pages/api/v1/progressReport/addReportRating.ts

```
@@ -38,9 +39,9 @@
 
 		const user_id = user?.id?.toString();
 
-		const { postId, proposalType, rating } = req.body;
+		const { postId, proposalType, rating, reportId } = req.body;
 
-		if (!proposalType || rating === undefined || isNaN(postId) || !isProposalTypeValid(proposalType)) {
+		if (!proposalType || rating === undefined || !reportId || isNaN(postId) || !isProposalTypeValid(proposalType)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)